plugins/ccgx/fu-ccgx-dmc-device.c

```diff
@@ -616,17 +616,22 @@ fu_ccgx_dmc_device_attach(FuDevice *device, FuProgress *progress, GError **error
 	if (fu_device_get_update_state(self) != FWUPD_UPDATE_STATE_SUCCESS)
 		return TRUE;
 
+	if (manual_replug == FALSE)
+		fu_device_add_flag(device, FWUPD_DEVICE_FLAG_WAIT_FOR_REPLUG);
+
```

Comment:
I think it's better to move this as an else clause below by the other manual_replug check so it's more obvious what is different for manual replug

Reply:
@superm1 
Could you check it at [0e64b65](https://github.com/fwupd/fwupd/pull/4563/commits/0e64b65ac32b78bca7ca98dbb0912f364db0b382) ?